packages/web3-eth-ens/package.json

```diff
@@ -2,12 +2,12 @@
     "name": "web3-eth-ens",
     "version": "1.0.0-beta.37",
```

Comment:
Is this correct? Shouldn't all the packages be released as `1.0.0`?

Reply:
Versions should not be bumped by hand. One should use `npx lerna publish` oder `npx lerna version`.